src/components/modals/index.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.  
  * Licensed under the MIT License.
  */
-import ActionCreatorEditor from './ActionCreatorEditor'
+import ActionCreatorEditor from './ActionCreatorEditor/ActionCreatorEditor'
```

Comment:
I would see if you can avoid the repeating pathing. I thought you wouldn't need this since you added the default export.

Reply:
Good catch.  (vscode auto change when file moved)